skllm/models/gpt/gpt_few_shot_clf.py

```diff
@@ -69,3 +70,104 @@ def _get_prompt(self, x: str) -> str:
         return build_few_shot_prompt_slc(
             x=x, training_data=training_data_str, labels=repr(self.classes_)
         )
+
+
+class MultiLabelFewShotGPTClassifier(_BaseZeroShotGPTClassifier):
```

Comment:
Would it be easier to inherit this from few shot classifier instead of base zero shot ? I feel like this would slightly improve the amount of redundant code.

Reply:
I tried, we can remove the redundant `fit` call. but the type annotation for `y` in `fit` method would be List[List['str']]. I have kept it as it is for now. Please suggest how to proceed.

Reply:
I see. The type hints might indeed be a problem. I think it is ok to keep the redundant code for now, as we are planning to rework the hierarchy of classes in v1 anyway, so we will deal with it then.